### DISCUSSION TITLE
[Projects Beta] Allow sorting issues by age

### DISCUSSION THREAD

**Discussion by jmtsi** [community/community#6701] [Projects and Issues]
It'd be great if I could sort my table/board of issues by their creation or modification date. The issues already have this data, so they could just be exposed as autofilled fields.

**Comment by manasesjesus** [comment]
The table/board of issues already have a **Sort** option to accomplish that: Newest, Oldest, Recently updated...

<img width="322" alt="issues" src="https://user-images.githubusercontent.com/24204142/138724121-28265ae5-0f36-470f-a97b-a35d671d2f77.png">

**Reply by jmtsi** [reply]
I meant the table and kanban-board views in the new [Projects Beta](https://github.com/features/issues/)

**Comment by jeffpaul** [comment]
Probably worth closing this in favor of #8518?
